tools/depends/native/Mako/Makefile

```diff
@@ -1,20 +1,18 @@
 include ../../Makefile.include
 PREFIX=$(NATIVEPREFIX)
 PLATFORM=$(NATIVEPLATFORM)
-DEPS= ../../Makefile.include Makefile
+DEPS = ../../Makefile.include Makefile ../../download-files.include
```

Comment:
I only just thought of this, but maybe we should have just added the download-files.include into the Makefile.include.
Dont take this as anything more than a passing comment, really not worth changing now

Reply:
won't work in cases where no Makefile.include exists ;)